lib/framework/elastic_connector_actions.rb

```diff
@@ -12,6 +12,7 @@
 module Framework
   class ElasticConnectorActions
     CONNECTORS_INDEX = '.elastic-connectors'
+    JOB_INDEX = '.elastic-connectors-sync-logs'
```

Comment:
I thought the index was called `.elastic-connectors-sync-jobs`? We can go with this name as well, but need to be consistent.